cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -1,75 +1,60 @@
 NAME
-       ballerina-build - Build the Ballerina package or standalone file and generate an executable
+       ballerina-build - Compiles the current package
 
 SYNOPSIS
-       bal build
-       bal build <ballerina-file-path>
-       bal build <ballerina-package-path>
-       bal build [--target-dir] <target-directory-path>
-       bal build [--offline] [--experimental] [-o | --output] <output-path> <ballerina-file-path>
-       bal build [--offline] [--experimental] [--cloud] [--observability-included]
-                 [--list-conflicted-classes] <ballerina-package-path>
+       bal build [OPTIONS] [<package>|<source-file>]
 
 
 DESCRIPTION
-       Compile Ballerina sources and produce the executable JAR file. However,
-       if the package does not contain an entry point, it will produce a non-executable
-       JAR file, which can be used in another package/program.
+       Compile the current package and its dependencies to produce a
+       standalone executable JAR (Java Archive) file.
 
-       Building a Ballerina package creates the executable `.jar` file
-       in the `<package-root>/target/bin` directory.
+       Build creates the resulting executable to the `target/bin` directory
+       when compiling the current package.
 
-       Building a standalone Ballerina file will create an executable `.jar` file in the
-       current directory. The name of the executable file will be
-       <ballerina-file-name>.jar'.
+       Build creates the resulting executable in the current directory
+       when compiling a single .bal file. The generated executable is named
+       after the source file.
 
-       Note: Building `.bal` files, which belong to a Ballerina package is not allowed.
+       Note: Building individual '.bal' files of the current package is not allowed.
```

Comment:
Do we need to say "current" here? Wondering if this could imply we could run a bal file of an arbitrary package like

```cmd
$ bal build foo/main.bal
```

Reply:
Agreed. Updated